docs/topics/js/js-to-kotlin-interop.md

```diff
@@ -156,35 +156,81 @@ C.Companion.callNonStatic(); // The only way it works
-| `BooleanArray`                                                   | `Int8Array`               | Carries the property `$type$ == "BooleanArray"`.                                           |
-| `List`, `MutableList`                                            | `KtList`, `KtMutableList` | Exposes an `Array` via `KtList.asJsReadonlyArrayView` or `KtMutableList.asJsArrayView`.    |
-| `Map`, `MutableMap`                                              | `KtMap`, `KtMutableMap`   | Exposes an ES2015 `Map` via `KtMap.asJsReadonlyMapView` or `KtMutableMap.asJsMapView`.     |
-| `Set`, `MutableSet`                                              | `KtSet`, `KtMutableSet`   | Exposes an ES2015 `Set` via `KtSet.asJsReadonlySetView` or `KtMutableSet.asJsSetView`.     |
-| `Unit`                                                           | Undefined                 | Exportable when used as return type, but not when used as parameter type.                  |
-| `Any`                                                            | `Object`                  |                                                                                            |
-| `Throwable`                                                      | `Error`                   |                                                                                            |
-| `enum class Type`                                                | `Type`                    | Enum entries are exposed as static class properties (`Type.ENTRY`).                        |
-| Nullable `Type?`                                                 | `Type | null | undefined` |                                                                                            |
-| All other Kotlin types, except for those marked with `@JsExport` | Not supported             | Includes Kotlin's [unsigned integer types](unsigned-integer-types.md).                     |
+| Kotlin                                                           | JavaScript                | Comments                                                                                |
+|------------------------------------------------------------------|---------------------------|-----------------------------------------------------------------------------------------|
+| `Byte`, `Short`, `Int`, `Float`, `Double`                        | `Number`                  |                                                                                         |
+| `Char`                                                           | `Number`                  | The number represents the character's code.                                             |
+| `Long`                                                           | `BigInt`                  | It works only with the `-Xes-long-as-bigint` flag turned on.                                                |
```

Comment:
```suggestion
| `Long`                                                           | `BigInt`                  | Needs the `-Xes-long-as-bigint` compiler option to be configured.                                                |
```
"Compiler option" rather than "flag".